straxen/online_monitor.py

```diff
@@ -0,0 +1,31 @@
+from straxen.rundb import get_mongo_url, default_mongo_dbname
+from strax import MongoFrontend, exporter
+from socket import getfqdn
+
+export, __all__ = exporter()
+
+default_online_collection = 'online_monitor'

```

Comment:
There is no collection for this yet. At the moment it is envisaged as a TTL with a duration of about a week, indexed by run-number.

Reply:
Which DB do we envision this going into?

Reply:
RunsDB makes most sense. It is accesable if needed via dali without the need to share DAQ database-cridentials.

Reply:
DAQ DB credentials don't work outside of the DAQ anyway.